### ISSUE TITLE
beaconManager.isBound() does not work in Android O

### ISSUE THREAD

**Issue by mgsteinkamp** [AltBeacon/android-beacon-library#613]
### Expected behavior
After onBeaconServiceConnect(), beaconManager.isBound() will evaluate as true
### Actual behavior
After onBeaconServiceConnect(), beaconManager.isBound() evaluates as false
### Steps to reproduce this behavior
```
beaconManager.bind(BeaconConsumer b);
...
onBeaconServiceConnect();
...
Log.i(TAG, "Bound: " + beaconManager.isBound());
```

### Mobile device model and OS version
Nexus 5X

### Android Beacon Library version
2.12.3

###Notes:
Worked totally fine on test devices from Android 5-7.1
Possibly related: RegionBootstrap doesn't seem to function on the first set any more either, but sorry, I don't have time to file a separate report.

**Comment by davidgyoung** [comment]
Thanks for this report.  A proposed solution is described in #600

**Comment by davidgyoung** [comment]
Fixed in #655
